### ISSUE TITLE
Primary Navigation Not Accessible Using Voiceover

### ISSUE THREAD

**Issue by jermartin77** [bigcommerce/cornerstone#1558]
Safari is a popular browser with screen readers. Navigating to these menus using the voiceover activation keys does not work. (https://webaim.org/articles/voiceover/). Many of our clients are being required to comply with these guidelines or threatened with litigation for not having acccessible websites. 

### Expected behavior
Primary navigation is navigable by a screenreader in safari at desktop using Voiceover. 

### Actual behavior
Primary navigation is skipped by screen reader at desktop in safari using Voiceover

### Steps to reproduce behavior
Open a cornerstone theme in Safari at desktop. Enable the screen reader (CMD+F5). Navigate to the menu. it is not announced. There is not a way to navigate the to the menu. This is because the `navPages-container` has the property `aria-hidden="true"` this works at mobile but hides the menu for desktop users with screenreaders.

**Comment by tylerrzaneray** [comment]
The accessibility shortcomings of this navigation are not limited to VoiceOver/Safari. Using other assistive technology and operating systems the navigation items are focusable but not visible to assistive technology, so nothing is announced for the nav items, navigation dropdowns cannot be opened, and the main nav items cannot be clicked to navigate to new pages.

Changing the `aria-hidden` attribute on `#menu` to `false` whenever the mobile menu is not used should solve this issue.

**Comment by bookernath** [comment]
Thanks for the report. We welcome PRs to help improve the accessibility of Cornerstone, and the theme is of course customizable to meet the needs of any particular implementation.

We'll consider holistic improvements to the accessibility of the theme in future versions.

**Comment by bookernath** [comment]
Fixed in https://github.com/bigcommerce/cornerstone/pull/1563
